packages/components/src/util/URLResolver.ts

```diff
@@ -305,7 +305,7 @@ export class URLResolver {
                 const url = row.get('url');
                 if (url) {
                     const params = ActionURL.getParameters(url);
-                    return AppURL.create('q', 'core', 'siteusers', params.userId);
+                    return AppURL.create('q', 'core', 'users', params.userId);
```

Comment:
See comment from the release notes section above. What was the scenario / use-case for this change (just curious)?